docs/rfcs/xxxx-invisible-minimal-sync.md

```diff
@@ -0,0 +1,91 @@
+# Invisible minimal sync
+
+## 0 - Basic considerations
+
+Considering the following :
+
+1. Alice creates a file in location `/foo/big.txt`
+2. Alice write a big amount of data into `/foo/big.txt`
+3. Alice create another file in location `/foo/small.txt`
+4. Bob wants to access to `/foo/small.txt` asap
+
+Here the simplest way to synchronize the data is:
+
+1. Sync `/foo/small.txt`'s file manifest
+2. Sync `/foo/big.txt`'s file manifest
+3. Sync `/foo`'s folder manifest
+
+However this is bad because `/foo`'s changes cannot be accessed by Bob as long as
+`/foo/big.txt` is not done with synchronization.
```

Comment:
```suggestion
However this is not ideal because `/foo`'s changes (included in `/foo`'s folder manifest) cannot be accessed by Bob as long as `/foo/big.txt` is not done with synchronization.
```